src/stores/FiltersHistoryStore.ts

```diff
@@ -0,0 +1,147 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import { observable, action, toJS } from 'mobx';
+import isEqual from 'lodash.isequal';
+import { IndexedDB, IndexedDbStores, indexedDbLimits } from '../api/indexedDb';
+import { SearchPanelType } from '../components/search-panel/SearchPanel';
+import {
+	EventFilterState,
+	MessageFilterState,
+	FilterState,
+} from '../components/search-panel/SearchPanelFilters';
+
+function getEquilizedFilterState(filter: FilterState) {
```

Comment:
Equilized is probably misspelled, if this function suppose to remove empty values, maybe we should call it filterNonEmptyFilters or something like that?